utils/iio_info.c

```diff
@@ -117,19 +120,36 @@ static void print_attr(const struct iio_attr *attr,
 
 	buf[BUF_SIZE - 1] = '\0';
 	value = iio_attr_get_static_value(attr);
-	if (!value) {
-		ret = iio_attr_read_raw(attr, buf, sizeof(buf) - 1);
-		if (ret < 0)
-			iio_strerror((int)ret, buf, sizeof(buf));
-		value = buf;
-	}
-	
-	if (ret >= 0)
+	if (value) {
 		printf(" value: %s\n", value);
-	else if (colors)
-		print_fmt(" value: " FMT_ERR "\n", value);
-	else
-		printf(" value: ERROR: %s\n", value);
+	} else {
+		ret = iio_attr_get_available(attr, &available_list, &available_count);
+		if (ret == 0) {
+			printf(" value:");
```

Comment:
Instead of "value:", a better label might be "options:". This clarifies to the user that the attribute has multiple possible values, not just a single one.